src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
```

Comment:
Javadoc comment is empty. Consider adding meaningful documentation. (Severity: low)